ra/ra_test.go

```diff
@@ -2349,20 +2379,37 @@ func TestNewOrderReuseInvalidAuthz(t *testing.T) {
 	// It should have an ID
 	test.AssertNotNil(t, order.Id, "Initial order had a nil ID")
 	// It should have one authorization
-	test.AssertEquals(t, len(order.Authorizations), 1)
+	if features.Enabled(features.NewAuthorizationSchema) {
+		test.AssertEquals(t, len(order.V2Authorizations), 1)
+	} else {
+		test.AssertEquals(t, len(order.Authorizations), 1)
+	}
 
 	// Fetch the full authz by the ID
```

Comment:
This comment is now in the wrong place. It should be below in the `else` clause. Also, maybe we can delete that `GetAuthorization` call (and comment) altogether?